presto-main/src/main/java/com/facebook/presto/sql/rewrite/MaterializedViewOptimizationRewrite.java

```diff
@@ -87,4 +99,41 @@ protected Node visitQuery(Query query, Void context)
             return query;
         }
     }
+
+    private static Query optimizeQueryUsingMaterializedView(
```

Comment:
Had only a quick look, wondering why did we remove utils class and moved this to a private method?

Reply:
AFAICT the utils class was not doing anything, so just to increase succintness/readability.